external/libseccomp.cmake

```diff
@@ -20,7 +20,7 @@ IF(NOT DEFINED LIBSECCOMP_BUILD_OWN OR LIBSECCOMP_BUILD_OWN STREQUAL "NO")
         bash -c "
             exe=`mktemp`
             echo -e '#include<stdio.h>\n#include<seccomp.h>\nint main(){printf(\"%d.%d\",SCMP_VER_MAJOR,SCMP_VER_MINOR);}' \
-                | gcc -I ${libseccomp_INC_PATH} -xc /dev/stdin -o $exe && $exe
+                | gcc -I ${libseccomp_INC_PATH} -xc /dev/stdin -o $exe >/dev/null 2>&1 && $exe
```

Comment:
A co jak się coś wysypie przy tym testowym programie, chociaż użytkownik jest przekonany, że systemowy seccomp jest dobry?

Reply:
Irytowały mnie błędy pojawiające się, kiedy libseccomp nie był znajdowany. Można wsadzć to wywołanie w ifa sprawdzającego, czy libseccomp_LIB_PATH/libseccomp_INC_PATH zostały znalezione.

Chy chodzi ci o to, żeby dodać -DLIBSECCOMP_SKIP_VERSION_CHECK?

Reply:
Raczej, że jeśli nie znalazł wogóle seccompa to niech nie sprawdza / nie wypisuje.
Natomiast jak znalazł i stwierdził, że jest zły, to chciałbym wiedzieć dlaczego.

Reply:
A, ok, to otoczę w ifa i dam ładniejszy output (zła wersja / nie udało się uruchomić/skompilować programu testowego, stodut:foo, stderr:bar).